src/services/remoteService.ts

```diff
@@ -177,11 +178,12 @@ export class RemoteService extends Disposable implements IRemoteService {
         await SSHConfiguration.saveGitpodSSHConfig(gitpodConfig);
     }
 
-    private getHostSSHConfig(host: string, launcher: string, proxyScript: string, extIpcPort: number, logLevel:string) {
+    private getHostSSHConfig(host: string, launcher: string, proxyScript: string, extIpcPort: number, logLevel: string) {
+        const extraArgs = (process.versions['electron'] && process.versions['microsoft-build']) ? '--ms-enable-electron-run-as-node' : '';
```

Comment:
should not we rather rely on vscode.env.uriScheme === 'vscode' || vscode.env.uriScheme === 'vscode-insiders'?  and encapsualte it somewhere?

I'm not sure about stability of `microsoft-build` version

Reply:
this check is done in vscode source code too so it should be stable I guess

Reply:
What is drawback not relying on it but on stable uriScheme?

Reply:
I think both are reliable, though uriScheme can be changed in product.json manually